src/main/java/gregtech/client/renderer/texture/cube/OrientedOverlayRenderer.java

```diff
@@ -125,11 +129,18 @@ public void registerIcons(TextureMap textureMap) {
             // require the normal texture to get the rest
             if (normalSprite == null) continue;
 
+            foundTexture = true;
+
             // normal
 
             final String active = String.format("%s_active", overlayPath);
             TextureAtlasSprite activeSprite = ICubeRenderer.getResource(textureMap, modID, active);
 
+            if (activeSprite == null) {
+                FMLClientHandler.instance().trackMissingTexture(new ResourceLocation(modID, "blocks/" + basePath + "/overlay_OVERLAY_FACE_active"));
```

Comment:
You could go one step further and add a `toString` to OverlayFace, and then substitute in the value of the face as a string in this warning, instead of going `OVERLAY_FACE`